code/modules/power/turbine.dm

```diff
@@ -167,10 +167,6 @@
 		return
 	if (usr.is_mob_incapacitated(TRUE))
 		return
-	if (!(ishuman(usr) || ticker) && ticker.mode.name != "monkey")
```

Comment:
Won't removing the `ishuman(usr)` check allow every mob type to start the turbine ?

Reply:
Good catch I added some more checks.

Reply:
This check is an outdated version of `usr.IsAdvancedToolUser()` anyway, fyi.